.ci/vfio_jenkins_job_build.sh

```diff
@@ -73,7 +73,6 @@ create_user_data() {
 		fi
 	fi
 
-	test_repo="github.com/kata-containers/tests"
```

Comment:
@chavafg ^^

Reply:
`.ci/vfio_jenkins_job_build.sh`  sources `lib.sh` in line 20. However, lib.sh exports `tests_repo` and here the variable is named `test_repo` (singular).

Good catch @chavafg , let me fix it.

Reply:
oh, so this is actually related to this change. @wainersm can you undo this change? This script runs before cloning the tests repo, so we need to have this one.

Reply:
ohh right, so maybe we can change that from `test_repo` to `tests_repo`

Reply:
Sent the fix, let's see if vfio job passes now.

Reply:
it passed :wink: